app/cdap/components/ConnectionList/Components/TabLabelCanBrowse/__tests__/TabLabelCanBrowse.test.tsx

```diff
@@ -29,7 +29,7 @@ describe('Test TabLabelCanBrowse Component', () => {
         index={0}
       />
     );
-    const ele = screen.getByTestId(/connections-tab-label-browse/i);
+    const ele = screen.getByTestId(/connections-tab-label-browse0/i);
```

Comment:
we should not have any changes in this file as it is not part of import schema feature

Reply:
done